src/main/java/com/streamingplatform/moviestreamingplatform/model/Movie.java

```diff
@@ -19,27 +28,35 @@
 
 @Entity
 @Table(name = "movie")
-@Getter @Setter @NoArgsConstructor
+@Getter
+@Setter
+@NoArgsConstructor
 public class Movie {
+
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     @Column(name = "id")
```

Comment:
@Ruleabogdan please remove column annotations that are identical to the java name

Reply:
this comment is for the entire project